src/lib/VTXSPI/devVTXSPI.cpp

```diff
@@ -23,8 +26,15 @@
 
 #define POWER_AMP_ON                            0b00000100111110111111
 #define POWER_AMP_OFF                           0x00
+#if defined(PLATFORM_ESP32)
+// ESP32 DAC pins are 0-255
+#define MIN_PWM                                 63  // Testing required.
+#define MAX_PWM                                 225 // Absolute max is 255.  But above 219 does nothing.
```

Comment:
This may need increasing for fine control.  But first we will need some hardware to test.